[docs] add basic code snippets auto-collapse function (#27495)
expo/expo: docs/README.md

<Video file="guides/color-schemes.mp4" />;
```

### Add code block

> Code blocks are a great way to add code snippets to our docs. We leverage the usuall code block Markdown syntax, but it's expanded to support code block titles, and an additional params.

Reviewer: ```suggestion
Code blocks are a great way to add code snippets to our docs. We leverage the usual code block Markdown syntax, but it's expanded to support code block titles and additional params.
```
Let's fix typos here

<!-- prettier-ignore -->
```mdx
    {/* For plain code block the syntax is unchanged (but we recommend to always add a title to the snippet): */}
    ```js
    // Your code goes in here
    ```

>     {/* To add a title, just enter it right after language, in the code block starting line: */}

Reviewer: ```suggestion
    {/* To add a title, enter it right after the language, in the code block starting line: */}
```

    ```js myFile.js
    // Your code goes in here
    ```
    ```js Title for a code block
    // Your code goes in here
    ```

    {/* Title and params can be separated by pipe ("|") characters, but they also work for block without a title: */}
    ```js myFile.js|collapseHeight=600
    // Your code goes in here
    ```
    ```js collapseHeight=200
    // Your code goes in here
    ```
```

#### Supported additional params

| Param            | Type   | Description                                                                                                                                                            |
| ---------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `collapseHeight` | number | The custom height to which code block should be automatically collapsed, the default value is `408` and it's applied unless `collapseHeight` param has been specified. |

Reviewer: ```suggestion
| `collapseHeight` | number | The custom height that the code block uses to collapse automatically. The default value is `408` and is applied unless the `collapseHeight` param has been specified. |
```
